client/components/dashboard/dashboard-service.js

```diff
@@ -79,7 +79,7 @@ const WidgetType = explorer.models.WidgetType;
 explorer.components.dashboard.DashboardService = function(arrayUtilService,
     errorService, widgetFactoryService, dashboardDataService,
     queryBuilderService,  dashboardVersionService, configService,
-    $filter, $location, $rootScope, $timeout, $window) {
+    explorerStateService, $filter, $location, $rootScope, $timeout, $window) {
```

Comment:
Arg missing from docstring?

That's a rather long arg list, would be nice to be able to use named args or an initializer dict. OK to leave as is also.